data-prepper-plugins/kafka-plugins/build.gradle

```diff
@@ -29,16 +29,18 @@ dependencies {
     implementation project(':data-prepper-plugins:buffer-common')
     implementation project(':data-prepper-plugins:blocking-buffer')
     implementation project(':data-prepper-plugins:aws-plugin-api')
+    // bump io.confluent:* dependencies correspondingly when bumping org.apache.kafka.*
+    // https://docs.confluent.io/platform/current/release-notes/index.html
     implementation 'org.apache.kafka:kafka-clients:3.6.1'
     implementation 'org.apache.kafka:connect-json:3.6.1'
     implementation project(':data-prepper-plugins:http-common')
     implementation libs.avro.core
     implementation 'com.fasterxml.jackson.core:jackson-databind'
     implementation 'io.micrometer:micrometer-core'
     implementation libs.commons.lang3
-    implementation 'io.confluent:kafka-avro-serializer:7.4.0'
-    implementation 'io.confluent:kafka-json-schema-serializer:7.4.0'
-    implementation 'io.confluent:kafka-schema-registry-client:7.4.0'
+    implementation 'io.confluent:kafka-avro-serializer:7.6.0'
```

Comment:
Is 7.6.1 available in the Confluent Maven repository?

Reply:
Not yet: https://mvnrepository.com/artifact/io.confluent/kafka-schema-registry-client?repo=confluent-packages